virt/about_virt/virt-architecture.adoc

```diff
@@ -34,6 +33,4 @@ include::modules/virt-about-cluster-network-addons-operator.adoc[leveloffset=+1]
 
 include::modules/virt-about-hpp-operator.adoc[leveloffset=+1]
 
-include::modules/virt-about-ssp-operator.adoc[leveloffset=+1]
```

Comment:
I think there is a cross reference to this module in another doc (virt-configuring-live-migration-limits.adoc), so removing this is breaking your build.

asciidoctor: WARNING: includes/virt-configuring-live-migration-limits.adoc: line 7: section title out of sequence: expected level 3, got level 4
Unknown ID or title "virt-about-ssp-operator_virt-architecture", used as an internal cross reference
>>> Validation of book virt in openshift-enterprise failed <<<